dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/discovery/SearchFacetFilter.java

```diff
@@ -344,6 +348,19 @@ public void addBody(Body body) throws SAXException, WingException, UIException,
         }
     }
 
+    private void updateQueryResultsAndOffset(Request request, DSpaceObject dso) {
```

Comment:
Perhaps reformat this method to follow the DSpace [coding conventions](https://wiki.duraspace.org/display/DSPACE/Code+Contribution+Guidelines#CodeContributionGuidelines-CodingConventions).